### ISSUE TITLE
x/website: discoverability of Security contents

### ISSUE THREAD

**Issue by Deleplace** [golang/go#57972]
There exists a nice page https://go.dev/security/ but I can't find how to navigate there from the website. As a result, some great contents like [Go Fuzzing](https://go.dev/security/fuzz/) seem to be buried.

We may add a Security link in the footer, and in https://go.dev/doc/

**Comment by findleyr** [comment]
A Security link in the footer makes sense to me. CC @julieqiu @jamalc

**Comment by julieqiu** [comment]
Agreed! It also looks like the `Why Go` dropdown currently links to https://go.dev/security/policy, and it should probably link to https://go.dev/security instead.

<img width="429" alt="Screenshot 2023-01-27 at 12 43 21 PM" src="https://user-images.githubusercontent.com/5943521/215157090-98888cbc-0e3a-4c9b-a236-9495a047d8bc.png">

**Comment by Deleplace** [comment]
@julieqiu as having 2 pages with similar terms can be a bit confusing, we must make sure to keep the link text consistent with the link URL.

Security -> https://go.dev/security/
Security Policy -> https://go.dev/security/policy

I agree that the former is more general, and provides a prominent access to the latter as a sub-content. So indeed it would make sense to amend the dropdown in order to link to the general "Security" page. The text "How Go can help keep you secure by default" fits well the general "Security" page and can be left unchanged.

**Comment by annagroebel** [comment]
Hi @Deleplace @julieqiu I guess die Issue can be closed, as the patch has been applied. 

> Change https://go.dev/cl/483777 mentions this issue: `_content: change nav dropdown text to "Security"`

**Comment by annagroebel** [comment]
> @annagroebel it looks like that change has not been sync'ed to https://pkg.go.dev.

Ah okay. I will fix it. Thank you!

**Comment by findleyr** [comment]
@annagroebel thank you!

**Comment by annagroebel** [comment]
> Change https://go.dev/cl/641715 mentions this issue: `feat: changes link in navigation to security documentation instead of sub`

Hi @findleyr I made the changes to the pkgsite repo. Could you please review it?
